api/swagger/server/restapi/resource/swagger.yml

```diff
@@ -861,6 +861,13 @@ paths:
           required: true
           x-nullable: false
           description: URL without dot (.), upper case and special characters
```

Comment:
can you update this description with the corresponding allowed names format?

Reply:
its [A-Za-z0-9]

Reply:
Shouldn't it be `[a-z0-9]` ? 
The description says "without upper case"

Reply:
There is a kind of validation on MassaStation side : https://github.com/massalabs/thyra/pull/718/files#diff-3e09e4ebd760f825d1cf5194f769c18c2b1d1c5f88e47e9ab74bea2498de3ebdL848 
I guess Swagger will block any request with `name` that doesn't match that Regex

And, I don't think we want to allow users to use uppercase in their website names. Otherwise, we will endup with:
- `Abc.massa`
- `abc.massa`
- `aBc.massa`
- ...